array_reverse/__tests__/array-reverse-test.js

```diff
@@ -0,0 +1,7 @@
+const { arrayReverser } = require('../array-reverse.js');
+
+describe('reverses arrays', () => {
+  it('makes arrays backwards', () => {
+    expect(arrayReverser([1, 2, 3])).toEqual[3, 2, 1];
+  });
+});
```

Comment:
This is a good test! In the future, I'd love to see more cases-- what happens with a negative number at the start or end of a given array? Maybe it keeps functioning perfectly, but maybe it does a weird thing! Only way to check is a test :)